code/modules/mob/new_player/new_player.dm

```diff
@@ -332,6 +332,9 @@
 		data_core.manifest_inject(humanc)
 		AnnounceArrival(humanc, rank)
 		AddEmploymentContract(humanc)
+		if(highlander)

```

Comment:
this should be a new gamemode datum function user_post_arrival(humanc)

then override the highlander gamemode datum to make them scottish